src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateField.const.ts

```diff
@@ -43,3 +42,9 @@ export const DisplayModeOptions: Record<
   [ItemResponseType.TouchTest]: undefined,
   [ItemResponseType.Video]: undefined,
 };
+
+export const KEYWORDS = {
+  SENTENCE: 'sentence',
+  ITEM_RESPONSE: 'item_response',
+  LINE_BREAK: 'line_break',
+} as const;
```

Comment:
Appreciate your creating constants for these strings!

One nit: We tend to use enums in the code for this sort of thing. Constants are fine, too, but this just seemed like a perfect opportunity to use an enum.

Secondly, I noticed you're using the `SENTENCE` value for two distinct purposes in the code: as an **item type** (`sentence`/`item_response`/`line_break`), as well as a **display mode value** (`sentence`/`bullet_list`/`sentence_option_row`/`sentence_row_option`/etc.).

It's an unfortunate coincidence that the keyword `sentence` is being used by the BE for two semantically distinct properties, and I think it's important we don't confuse the two purposes of that keyword. So I think two separate, meaningfully named enums would make more sense:

```suggestion
export enum FieldType {
  Sentence = 'sentence',
  ItemResponse = 'item_response',
  LineBreak = 'line_break',
}

export enum DisplayMode {
  Sentence = 'sentence',
  SentenceOptionRow = 'sentence_option_row',
  SentenceRowOption = 'sentence_row_option',
  BulletListOptionRow = 'bullet_list_option_row',
  BulletListTextRow = 'bullet_list_text_row',
}
```

Reply:
good comment I understand your point but there is a point of why in TS is preferable to use a const with the `as const` word instead of enum... for more information look at this https://www.youtube.com/watch?v=jjMbPt_H3RQ
but in general the recommendation is to avoid using enum inside ts it's transformed basically into a IIFE so my recommendation is to avoid using enums in typescript
I made a little playground with that, that's the same that Matt Pocock mentioned on his video
<img width="1985" alt="image" src="https://github.com/user-attachments/assets/b630ce14-7b81-405c-a721-74c0c4281684">

Reply:
Sure, `as const` is fine with me. Thanks for sharing the video!